language/ru.jsonc

```
@@ -682,173 +682,173 @@
-    "Default only",
+    "Только стандарт.",
 
     // Option for "Show emoji" setting where all emoji are shown.
-    "All",
+    "Все",
 
     // Softkey labels for importing the token (from a file) in the login screen. The long variant of the label is shown as the button text.
-    "Import",
-    "Import token",
+    "Импорт.",
+    "Импортировать токен",
 
     // Title text for the token import file picker screen.
-    "Select token file",
+    "Выбрать файл с токеном",
 
     // Text shown in loading screen when emoji data is being downloaded or updated.
-    "Downloading emoji...",
+    "Скачивание эмодзи...",
 
     // Related to "Lines shown in the about screen for which language" above.
-    "French",
-    "Bulgarian",
+    "Французский",
+    "Болгарский",
 
     // Settings option for toggling markdown text formatting support in chat (bold, italic, etc).
-    "Text formatting",
+    "Форматирование текста",
 
     // Softkey labels for inserting an emoji in the "send message" screen.
-    "Emoji",
-    "Insert emoji",
+    "Эмодзи",
+    "Вставить эмодзи",
 
     // Title for emoji picker screen.
-    "Select emoji",
+    "Выбрать эмодзи",
 
     // Prompt shown when opening the emoji picker and the emoji images have not been downloaded onto the device.
-    "Download emoji data?",
+    "Скачать эмодзи?",
 
     // Indicator shown at the end of messages which have been edited. If possible, this should be translated the same way that it's translated in the official Discord clients.
-    "(edited)",
+    "(изм.)",
 
     // Softkey command for inserting a '_' character in the token input screen. Some devices don't have a built-in way to insert a '_'.
-    "Underscore",
-    "Insert underscore",
+    "Подчеркивание",
+    "Вставить подчеркивание",
 
     // Related to "Lines shown in the about (credits) screen for what each developer" above.
-    "Notification sound",
+    "Звук уведомления",
 
     // Settings option for toggling vibration alert for notifications.
-    "Vibration",
+    "Вибрация",
 
     // Softkey command (short label and long label) for muting a channel/server/etc. or unmuting if it is already muted.
     // For technical reasons it's easier to have one label for "mute/unmute" rather than separate "Mute" and "Unmute" labels that change dynamically.
     // The short "Mute" label is also shown for unmuting. The long label can also be just "Mute" if "mute/unmute" doesn't make sense in your language.
-    "Mute",
-    "Mute/Unmute",
+    "Заглушить",
+    "Заглушить",
 
     // Hotkey actions for instantly moving to the top or bottom of the chat view. Related to "Names of hotkey actions shown in the key mapper" above.
     "scrolling to top",
     "scrolling to bottom",
 
     // Settings option for toggling whether to send the typing indicator, i.e. show other people that you are typing when you select "Send message" or "Upload file".
-    "Send typing indicator",
+    "Отпр. статус печатания",
```

```suggestion
    "Отправить статус печати",
```
...и всё равно звучит убого. Были бы экраны не 128 в ширину, можно было бы нормально `сообщать о наборе сообщения` написать...